server/src/logic.js

```diff
@@ -382,7 +382,7 @@ export const getHandlers = ({ models, creators: Creators }) => {
               return Promise.reject(Error('Invalid group!'));
             }
             return group.removeUser(user).then((rows) => {
-              if (rows) { return true; }
+              if (rows) { return Group.findById(groupId); }
```

Comment:
couldn't you just return the `group`. I assume `removeUser()` updates the internal state automatically. Maybe not but worth a try since it would save a DB query.
```
if (rows) { return group; }
```